inc/functions_upload.php

```diff
@@ -740,15 +743,9 @@ function upload_file($file, $path, $filename="")
 	$filename = preg_replace("#/$#", "", $filename); // Make the filename safe
 	$moved = @move_uploaded_file($file['tmp_name'], $path."/".$filename);
 
-	$moved_cdn = false;
-	$cdn_base_path = rtrim($mybb->settings['cdnpath'], '/');
-	$cdn_path = rtrim(realpath($cdn_base_path . '/' . $path), '/');
+	$cdn_path = '';
```

Comment:
If my acknowledgement of PHP is correct, this line is unnecessary because `$cdn_path` is passed by argument.